src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -179,8 +177,7 @@ public override void Loaded(ViewLoadedParams viewLoadedParams)
             this.ViewModel.DynamoView = viewLoadedParams.DynamoWindow;
         }
 
-
-        public void Shutdown()
+        public override void Shutdown()
```

Comment:
was the compiler complaining about method hiding?

Reply:
yes, there was a warning.